spec/features/site_customization/custom_pages_spec.rb

```diff
@@ -142,25 +134,125 @@
+                            title_en: "Title in English",
+                            title_es: "Titulo en Español",
+                            subtitle_en: "Subtitle in English",
+                            subtitle_es: "Subtitulo en Español",
+                            content_en: "Content in English",
+                            content_es: "Contenido en Español"
+                            ) }                          
 
-      scenario "Not listed in more information page due to different locale" do
-        custom_page = create(:site_customization_page, :published,
-          slug: "another-slug", title: "Ce texte est en français",
-          subtitle: "Subtitle for custom page",
-          more_info_flag: false,
-          locale: "fr"
-        )
+    background do
+      admin = create(:administrator)
+      login_as(admin.user)
+    end
 
-        visit help_path
+    before do
+      @edit_page_url = edit_admin_site_customization_page_path(custom_page)
+    end
 
-        expect(page).not_to have_content("Ce texte est en français")
+    scenario "Add a translation in Português", :js do
+      visit @edit_page_url
 
-        visit custom_page.url
+      select "Português", from: "translation_locale"
+      fill_in 'site_customization_page_title_pt_br', with: 'Titulo em Português'
 
-        expect(page).to have_title("Ce texte est en français")
-        expect(page).to have_selector("h1", text: "Ce texte est en français")
-        expect(page).to have_content("Subtitle for custom page")
+      click_button 'Update Custom page'
+      expect(page).to have_content "Page updated successfully"
+      
+      visit @edit_page_url
+      expect(page).to have_field('site_customization_page_title_en', with: 'Title in English')
+
+      click_link "Español"
+      expect(page).to have_field('site_customization_page_title_es', with: 'Titulo en Español')
+
+      click_link "Português"
+      expect(page).to have_field('site_customization_page_title_pt_br', with: 'Titulo em Português')
+    end
+
+    scenario "Update a translation", :js do
+      visit @edit_page_url
+
+      click_link "Español"
+      fill_in 'site_customization_page_title_es', with: 'Titulo correcta en Español'
+
+      click_button 'Update Custom page'
+      expect(page).to have_content "Page updated successfully"
+
+      visit custom_page.url
+
+      select('English', from: 'locale-switcher')
+      
+      expect(page).to have_content("Title in English")
+
+      select('Español', from: 'locale-switcher')
+ 
+      expect(page).to have_content("Titulo correcta en Español")
+    end
+
+    scenario "Remove a translation", :js do
+      visit @edit_page_url
+
+      click_link "Español"
+      click_link "Remove language"
+
+      expect(page).not_to have_link "Español"
+
+      click_button "Update Custom page"
+      visit @edit_page_url
+      expect(page).not_to have_link "Español"
+    end
+
+    context "Globalize javascript interface" do
```

Comment:
It looks like some of these tests could be replaced by `it_behaves_like "translatable"`. I'm not sure, though. @mlovic @papayalabs What do you think?

Reply:
Using it_behaves_like "translatable" will use test "Add a translation scenario" with "Français" and I think is important to test with "Português" because for this language has some particularities that may trigger some errors that will not trigger with other languages. But again is my perspective, if you guys prefer with "translatable" code I will changed ;)

Reply:
I agree it's important to test Brazilian Portuguese :+1:! In the `translatable` shared examples there's a scenario for that case, called "Add a translation for a locale with non-underscored name". Would that be enough, or do we need something specific to custom pages? If we need something specific, we could add `it_behaves_like "translatable"` and then add tests for specific cases, like in [collaborative legislation specs](https://github.com/AyuntamientoMadrid/consul/pull/1628/files#diff-8417d724f680660eb405aab8cda022adR106). What do you think?

Reply:
I had not seen that scenario in the translatable shared. I think that would be enough, Thanks for the clarification @javierm! I am going to updated with translatable shared

Reply:
Hi @javierm! in the ´translatable shared class´is there a way to pass some field that are not translatable? For example in cutom pages the default for ´status´is ´draft´and we need to pass ´status´´published´for tests.

Reply:
How about adding a new factory? In `spec/factories/administration.rb`:

```ruby
  factory :site_customization_page, class: 'SiteCustomization::Page' do
    slug "example-page"
   # (...)

    trait :published do
      status "published"
    end
    
    # (...)

    factory :published_site_customization_page, traits: [:published] # <---- Add this line
  end
```
Maybe that way we could pass `published_site_customization_page` as the factory name to `translatable`.

You might run into issues using the translatable behaviour because (so far) some of the specs there don't work well with CKEditor. If that happens, it's OK to leave the tests where they are for now.

Reply:
Thanks so much for the tip!!

Reply:
Was not necessary to create published_site_customization_page, the thing was that in translatable class it uses a ´update_button_text´ and for default uses the name ´Save changes´but the button in Custom Pages is ´Update Custom page´and I thought that the page was not published. I already add ´Update Custom page´ name in translatable class. But you are right does not recognize CKEditor fields nevertheless it works fine with the others fields and the tests now does not check CKEditor fields. So how I proceed? Leave tests as they are or use translatable with (title,subtitle) only? Thanks!!

Reply:
Leaving tests as they are is fine :ok_hand:. We'll try to better integrate CKEditor with Capybara in the future.

Reply:
Ok @javierm! So I thing all the requested changes are ready. I unified everything in one commit. Thanks!